feature/my/src/main/java/com/hankki/feature/my/myjogbo/MyJogboViewModel.kt

```diff
@@ -69,23 +87,27 @@ class MyJogboViewModel @Inject constructor(
     }
 
     fun deleteJogboStore() {
-        val deleteList = _myJogboState.value.myJogboItems.filter {
-            it.jogboSelected
-        }.map {
-            it.jogboId
-        }
-        viewModelScope.launch {
-            myRepository.deleteJogboStores(deleteList)
-                .onSuccess {
-                    _myJogboState.value = _myJogboState.value.copy(
-                        showDialog = false,
-                        editMode = false
-                    )
-                    getMyJogboList()
-                }
-                .onFailure { error ->
-                    Timber.e(error)
-                }
+        val currentState = _myJogboState.value.uiState
+        if (currentState is UiState.Success) {
+            val jogboItems = currentState.data
+            val deleteList = jogboItems.filter {
+                it.jogboSelected
+            }.map {
+                it.jogboId
+            }
+            viewModelScope.launch {
+                myRepository.deleteJogboStores(deleteList)
+                    .onSuccess {
+                        _myJogboState.value = _myJogboState.value.copy(
+                            showDialog = false,
+                            editMode = false
+                        )
+                        getMyJogboList()
+                    }
+                    .onFailure { error ->
+                        Timber.e(error)
+                    }
```

Comment:
놀랍게도 이것도 `::` 연산이 된다는것!
```suggestion
                    .onFailure(Timber::e)
```

Reply:
요건 몰랐네유 꿀팁감사합니다